.github/workflows/terraformChecks.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       TERRAFORM_DIRS: |
-          dev stg prod
```

Comment:
We have `prod-dmz` sprinkled throughout, but not here. I'm also not seeing the directory on `main`. `prod-dmz` should be another directory, like `prod,` yes? I'm guessing we haven't gotten to this yet and the expectation is the `prod-dmz` deploy will fail after `stg` but I wanted to point this out just in case.

Reply:
I appreciate you pointing this out! You're correct, we haven't gotten there just yet. Right now, all of our effort is being limited to the `dev` and `global` RGs, for the sake of cost savings. Once we can confirm that we have everything set and ready in `dev`, along with the deployment flow for the DIBBs components working properly, we can turn the other environments on and begin deploying that infrastructure.